src/shared/components/plots/WaterfallPlot.tsx

```diff
@@ -454,11 +461,18 @@ export default class WaterfallPlot<D extends IBaseWaterfallPlotData> extends Rea
     
     @bind
     private getChart() {
+        const svgViewBox = () => {
+            if (this.props.horizontal) {
+                return `0 -10 ${this.svgWidth} ${this.svgHeight}`;
+            }
+            return `0 0 ${this.svgWidth} ${this.svgHeight}`
+        };
         return (
             <div
                 ref={this.containerRef}
                 style={{width: this.svgWidth, height: this.svgHeight}}
             >
+                <p style={plotTitleStyles}>{this.title}</p>
```

Comment:
Why did you move the title out of the svg component? This may (I do not know) give a problem with export of the figure?